ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -461,12 +466,12 @@ open class SegmentedControl: UIControl {
         }
 
         if shouldSetEqualWidthForSegments {
-            if let windowWidth = window?.safeAreaLayoutGuide.layoutFrame.width {
-                if traitCollection.userInterfaceIdiom == .pad {
-                    maxButtonWidth = max(windowWidth / 2, 375.0)
-                } else {
-                    maxButtonWidth = windowWidth
-                }
+            let windowSafeAreaInsets = window.safeAreaInsets
+            let windowWidth = window.bounds.width - windowSafeAreaInsets.left - windowSafeAreaInsets.right
+            if traitCollection.userInterfaceIdiom == .pad {
+                maxButtonWidth = max(windowWidth / 2, 375.0)
```

Comment:
Nit: should there be a comment or a const somewhere explaining why you're potentially padding up to the magic number of 375?